spdx-utils/src/main/kotlin/SpdxUtils.kt

```diff
@@ -77,9 +77,16 @@ fun calculatePackageVerificationCode(files: List<File>, excludes: List<String> =
  * Return the SHA-1 sum of the given file as hex string.
  */
 private fun sha1sum(file: File): String {
-    file.inputStream().use {
+    file.inputStream().use { inputStream ->
+        val buffer = ByteArray(4 * 1024 * 1024)
         val digest = MessageDigest.getInstance("SHA-1")
-        return digest.digest(it.readBytes()).toHexString()
+
+        var length: Int
+        while (inputStream.read(buffer).also { length = it } >= 0) {
```

Comment:
You could reduce the check from `>=` to `>` as we don't need to do anything for 0 bytes read.

Reply:
Done.